pkg/controller/util/federated_updater.go

```diff
@@ -33,7 +33,8 @@ import (
 type FederatedOperationType string
 
 const (
-	OperationTypeAdd    = "add"
+	OperationTypeCreate = "create"
+	OperationTypeRead   = "read"
```

Comment:
Really don't want to have a read operation here - it dramatically enlarges the scope of the updater.  Hopefully this is unnecessary when version checks on namespaces in the host cluster are skipped.